docs/troubleshooting/updating-nitric.mdx

```diff
@@ -0,0 +1,18 @@
+## Updating Nitric
+
+If you're ready to upgrade your projects to a newer version of Nitric
+
+- Start by refreshing any versions of the membrane which are currently residing on Docker:
+  - Run `docker image rm nitricimages/membrane-local`
+  - Checkout the latest version of the membrane
+  - Run `make dev-docker` (or other target e.g. `make aws-docker`. Please refer to the [membrane readme](https://github.com/nitrictech/membrane/blob/main/README.md) for more details
+  - Update the local membrane with `docker pull nitricimages/membrane-local`
```

Comment:
I wondering if a quick update the `doctor` command could help with this instead. So when you run:
`nitric doctor`
We could internally run:
- `docker pull nitricimages/membrane-local`
- `docker pull nitricimages/membrane-aws`
- `docker pull nitricimages/membrane-gcp`
- `docker pull nitricimages/membrane-do`
- `docker pull nitricimages/membrane-azure`

**OR** (probably better)

We could just prune the contents of the `nitricimages` repo in the local docker environment, which would force the images to be pulled the next time they're used and avoid pulling images that aren't needed (like clouds that aren't in use)

Reply:
Breaking this out into a new issue against the CLI. 
We can change the docs to add this when complete.